system-tests/services/test-apps.js

```diff
@@ -306,7 +306,7 @@ describe("Services", function() {
         .get(".page-body-content table")
         .contains(serviceName, { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
         .should("exist")
-        .as("serviceName");
+        .as(serviceName);
```

Comment:
I'm pretty sure we need those quotes here. Especially when we still try getting the alias as `get("@serviceName")`

Reply:
@nLight correct, added those back